tests/integration/golang/mlflow/experiment/delete_test.go

```diff
@@ -40,6 +40,10 @@ func (s *DeleteExperimentTestSuite) SetupTest() {
 	s.fixtures = fixtures
 }
 
+func (s *DeleteExperimentTestSuite) TearDownTest() {
```

Comment:
this is not really important in tests.

Reply:
this is like with table names today =)

Reply:
alright, I have removed all the test bits!